Makefile

```diff
@@ -52,7 +52,7 @@ release: build
 	git push origin $${NEW_VERSION}
 
 test: fmtcheck vet test_deps
-	@$(GO_TEST) $(PACKAGE_LIST) -timeout=30s -parallel=4
+	@$(GO_TEST) $(PACKAGE_LIST) -timeout=30s -parallel=4 -coverprofile=coverage.out
```

Comment:
does this add considerably more time to the test run? if so, how about one target without coverage and another with?

Reply:
on my system it is ~.4s, with only testing the command once each lol. But I think your right and I'll make a `make coverage` or something for this specifically.